### PR TITLE
remove -<snappy> from version

### PR DESCRIPTION
**Pull request by Licenser**
having -<snappy> as part of the version makes it near impossibl to use this as a hex dependency as matchers won't match it.

### CODE REVIEW

**Comment by zmstone** [comment]
Thanks!
Could you please update README?
Remove the versioning section,
and add a snappyer version to google/snappy version mapping?
e.g.
At current state, it would be 1.0.x -> 1.1.3

File: src/snappyer.app.src [hunk 1]
```diff
@@ -1,7 +1,7 @@
 { application
 , snappyer
 , [ {description,"Snappy as nif for Erlang"}
-  , {vsn,"1.1.3-1.0.5"}
```
Comment (zmstone) [inline src/snappyer.app.src]: 1.1.3 is the google/snappy version.
please change it to 1.2.0 instead.
(it should be 1.0.6 as a semver bump, but I guess 1.2.0 is less confusing as it is alphabetically  sorted after 1.1.3)

Comment (zmstone) [inline src/snappyer.app.src]: also need to update version in app.src

Comment (zmstone) [inline src/snappyer.app.src]: also need to update version in Makefile

**Comment by Licenser** [comment]
sure thing!

**Comment by zmstone** [comment]
New version (1.2.0) published to hex. Please give it a try.

**Comment by Licenser** [comment]
Works like a charm! Thanks :D

**Pull request merged**